core/actions/operation.ts

```diff
@@ -323,6 +324,16 @@ export class Operation extends ActionBuilder<dataform.Operation> {
     return dataform.Target.create(this.proto.target);
   }
 
+  /** @hidden */
+  public getHasOutput() {
```

Comment:
Maybe specify a return type? I assume it is `boolean` but maybe it is better to make it clear?

Reply:
With most methods we just let the return value be implicit, but I'm also a fan of explicitly putting the returned types. Would be good to update everything else too!

Reply:
I think we can do it as we go, when touching some code.